### PR TITLE
Added ability to Query Org Vdc

### PR DESCRIPTION
**Pull request by funky81**
This PR adds:
- support to query for organization and vdc therefore we can get total resources (vCPU, RAM and Storage) for each Vdc within organization

Note: made based on vCloud Director 9.1 (vCloud API 30.0)

### CODE REVIEW

File: govcd/vdc.go [hunk 1]
```diff
@@ -23,13 +23,26 @@ type Vdc struct {
 	VApp   *types.VApp
 }
 
+type OrgVdcRecord struct {
```
Comment (dataclouder) [inline govcd/vdc.go]: This structure should not be needed for query-based operations

File: govcd/vdc.go [hunk 2]
```diff
@@ -23,13 +23,26 @@ type Vdc struct {
...
+
 func NewVdc(cli *Client) *Vdc {
 	return &Vdc{
 		Vdc:    new(types.Vdc),
 		client: cli,
 	}
 }
 
+// create instance with reference to types.OrgVdcRecord
+func NewOrgVdcRecord(cli *Client) *OrgVdcRecord {
```
Comment (dataclouder) [inline govcd/vdc.go]: Same as above. Only entities that are handled through direct API calls need this treatment.

**Comment by lvirbalas** [comment]
@funky81 Are you using go-vcloud-director library stand-alone (i.e. without Terraform)? Please share more about your use case, so we know the users better :)

File: README.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ To show the SDK in action run the example:
 ```
 mkdir ~/govcd_example
 go mod init govcd_example
-go get github.com/vmware/go-vcloud-director/v2@master
+go get github.com/funky81/go-vcloud-director/v2@master
```
[the remark this answers is not in the document]
Comment (funky81) [reply README.md]: Hi @didainius. I can removed that one, it accidentally insert in the pull request.

**Comment by funky81** [comment]
@lvirbalas my use case is not for terraform.
Currently I'm developing portal to accommodate / facilitate automation for my company. I take this library as part of Middleware to automate with other infrastructure (k8s, object storage, etc). Most of the cases will use this for deployment or change request.

**Comment by Didainius** [comment]
@funky81,
Sorry that it has left in limbo and we need to decide whether to merge this and potentially have latest master merged.

The PR got a little messy now because master is further away so I will start with the original intention:
* What is your end goal for this PR? That is what exactly are you missing in the current state of govcd.

This should help to discus the best approach and move it forwards finally.

Thanks!

**Comment by Didainius** [comment]
@funky81 ,
I am closing this for now. Feel free to re-open if you can address the above comments.

**Pull request closed without merging**